woeip/templates/includes/_header.html

```diff
@@ -0,0 +1,21 @@
+<!-- Navigation Bar -->
+<ul>
+    <li><h1><a href="{% url 'index' %}">West Oakland Air Quality</a></h1></li>
+    <li><a href="{% url 'upload' %}">File Upload</a></li>
+    <li>
+        {% if user.is_authenticated %}
+            {{user}}: <a href="{% url 'logout' %}">Logout</a>
+        {% else %}
+            Guest: <a href="{% url 'login' %}">Login</a>
+        {% endif %}
+    </li>
+    <li><a href="mailto:contact_email">Request Account</a></li>
```

Comment:
Should this be a valid email address?

Reply:
Soon. I'll open an issue to make sure we come back to it, once the WOEIP team has discussed it.